src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -242,3 +242,228 @@
+    type: group
+    short-summary: Probes gather information, such as utilization, to be evaluated by rules
+"""
+helps['network application-gateway rule'] = """
+    type: group
+    short-summary: Rules evaluate probe information and define routing rules
+"""
+helps['network application-gateway ssl-cert'] = """
+    type: group
+    short-summary: Commands to manage app gateway SSL certificates
+"""
+helps['network application-gateway url-path-map'] = """
+    type: group
+    short-summary: Commands to manage app gateway URL path maps
+"""
+helps['network application-gateway url-path-map rule'] = """
+    type: group
+    short-summary: Commands to manage app gateway URL path map rules
+"""
+helps['network dns'] = """
+    type: group
+    short-summary: Host your DNS domain in Azure
+"""
+helps['network dns record'] = """
+    type: group
+    short-summary: Commands to manage DNS records contained in a record-set
+"""
+helps['network dns record a'] = """
+    type: group
+    short-summary: Manage DNS A records
+"""
+helps['network dns record aaaa'] = """
+    type: group
+    short-summary: Manage DNS AAAA records
+"""
+helps['network dns record cname'] = """
+    type: group
+    short-summary: Manage DNS CNAME records
+"""
+helps['network dns record mx'] = """
+    type: group
+    short-summary: Manage DNS MX (mail) records
+"""
+helps['network dns record ns'] = """
+    type: group
+    short-summary: Manage DNS NS (nameserver) records
+"""
+helps['network dns record ptr'] = """
+    type: group
+    short-summary: Manage DNS PTR (pointer) records
+"""
+helps['network dns record srv'] = """
+    type: group
+    short-summary: Manage DNS SRV records
+"""
+helps['network dns record txt'] = """
+    type: group
+    short-summary: Manage DNS TXT records
+"""
+helps['network dns record-set'] = """
+    type: group
+    short-summary: Manage DNS record-set
+"""
+helps['network dns zone'] = """
+    type: group
+    short-summary: Commands to manage DNS zones
+"""
+helps['network express-route'] = """
+    type: group
+    short-summary: Dedicated private network fiber connections to Azure
+"""
+helps['network express-route circuit'] = """
+    type: group
+    short-summary: Commands to manage express-route circuits
+"""
+helps['network express-route circuit-auth'] = """
+    type: group
+    short-summary: Commands to manage express-route circuit authentication
+"""
+helps['network express-route circuit-peering'] = """
+    type: group
+    short-summary: Commands to manage express-route peering
+"""
+helps['network express-route service-provider'] = """
+    type: group
+    short-summary: Commands to view express-route service providers
+"""
+helps['network lb'] = """
+    type: group
+    short-summary: Deliver high availability and network performance to your applications
+"""
+helps['network lb address-pool'] = """
+    type: group
+    short-summary: Commands to manage LB address pools
```

Comment:
for the commands under the lb group, I figured the abbreviation is fine given the premium for help text space